production/cassandra.yml

```diff
@@ -16,6 +16,27 @@ apiVersion: v1
 kind: List
 items:
 - apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: jaeger-configuration
+    labels:
+      app: jaeger
+      jaeger-infra: configuration
+  data:
```

Comment:
Am I right that this create two files? `collector` and `query`

Reply:
Yes and no. Config Maps are key/value entries, where a key can contain other key/value entries (nested map). It can be consumed in several ways, and one of them is by mounting a key as a volume. In our case, we are mounting each top-level key as a file, so, yes, we have a collector.yaml and a query.yaml when we view it from the container.